src/pages/home/sidebar/ChatSwitcherOptionPropTypes.js

```diff
@@ -10,9 +10,15 @@ export default PropTypes.shape({
     // The URL of the person's avatar
     icon: PropTypes.string,
 
+    // URLS OF FIRST TWO PARTICIPANTS' AVATAR FOR GROUP CHAT
+    groupIcons: PropTypes.arrayOf(String),
+
     // The type of option we have e.g. user or report
     type: PropTypes.string,
 
+    // ARRAY OF THE LOGIN OF ALL USERS IN A CHAT
```

Comment:
same here

Reply:
Alright. Fix coming up in my next commit.